dask/dataframe/tests/test_groupby.py

```diff
@@ -2438,7 +2444,7 @@ def test_groupby_sort_true_split_out():
     not dd._compat.PANDAS_GT_100, reason="observed only supported for newer pandas"
 )
 @pytest.mark.parametrize("known_cats", [True, False])
-@pytest.mark.parametrize("ordered_cats", [True, False])
+@pytest.mark.parametrize("ordered_cats", [False])
```

Comment:
Why did we need to drop `True` here? Since there's only a single value for `ordered_cats` now, we can probably remove the corresponding `parametrize` and just inline the `False` inside the test

Reply:
That was an accidental change that I was doing while experimenting. Reverting...